backbone/src/main/java/org/researchstack/backbone/ui/ViewTaskActivity.java

```diff
@@ -91,6 +91,14 @@ public Step getCurrentStep()
         return currentStep;
     }
 
+    public int getContentViewId() {
```

Comment:
Nice to have: `@LayoutRes` annotation

Reply:
Yes thanks!  I was trying to remember the annotation for that and kept coming up with nothing.